[2024-12-06T15:39:21.656+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2024-12-06T15:39:21.667+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: insert_cleaned_data.set_vars manual__2024-12-06T15:37:35.362057+00:00 [queued]>
[2024-12-06T15:39:21.673+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: insert_cleaned_data.set_vars manual__2024-12-06T15:37:35.362057+00:00 [queued]>
[2024-12-06T15:39:21.674+0000] {taskinstance.py:2866} INFO - Starting attempt 1 of 4
[2024-12-06T15:39:21.684+0000] {taskinstance.py:2889} INFO - Executing <Task(PythonOperator): set_vars> on 2024-12-06 15:37:35.362057+00:00
[2024-12-06T15:39:21.690+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=3099) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-12-06T15:39:21.691+0000] {standard_task_runner.py:72} INFO - Started process 3100 to run task
[2024-12-06T15:39:21.691+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'insert_cleaned_data', 'set_vars', 'manual__2024-12-06T15:37:35.362057+00:00', '--job-id', '2012', '--raw', '--subdir', 'DAGS_FOLDER/insert_cleaned_data.py', '--cfg-path', '/tmp/tmph_0advbc']
[2024-12-06T15:39:21.692+0000] {standard_task_runner.py:105} INFO - Job 2012: Subtask set_vars
[2024-12-06T15:39:21.725+0000] {task_command.py:467} INFO - Running <TaskInstance: insert_cleaned_data.set_vars manual__2024-12-06T15:37:35.362057+00:00 [running]> on host cb0074367deb
[2024-12-06T15:39:21.785+0000] {taskinstance.py:3132} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='artzsh' AIRFLOW_CTX_DAG_ID='insert_cleaned_data' AIRFLOW_CTX_TASK_ID='set_vars' AIRFLOW_CTX_EXECUTION_DATE='2024-12-06T15:37:35.362057+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-12-06T15:37:35.362057+00:00'
[2024-12-06T15:39:21.786+0000] {taskinstance.py:731} INFO - ::endgroup::
[2024-12-06T15:39:21.807+0000] {crypto.py:82} WARNING - empty cryptography key - values will not be stored encrypted.
[2024-12-06T15:39:21.812+0000] {insert_cleaned_data.py:106} INFO - Обновлён dm_var до 2
[2024-12-06T15:39:21.812+0000] {python.py:240} INFO - Done. Returned value was: None
[2024-12-06T15:39:21.818+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2024-12-06T15:39:21.818+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=insert_cleaned_data, task_id=set_vars, run_id=manual__2024-12-06T15:37:35.362057+00:00, execution_date=20241206T153735, start_date=20241206T153921, end_date=20241206T153921
[2024-12-06T15:39:21.865+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2024-12-06T15:39:21.878+0000] {taskinstance.py:3895} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2024-12-06T15:39:21.880+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
